code/hash/update-vs-[]=.rb

```diff
@@ -0,0 +1,21 @@
+require 'benchmark/ips'
+
+ENUM = (1..100)
+
+def fast
+  ENUM.each_with_object({}) do |e, h|
+    h[e] = e
+  end
+end
+
+def slow
+  ENUM.each_with_object({}) do |e, h|
+    h.update(e => e)
```

Comment:
Here you create an additional Hash object which of curse will be much slower than only provide two "static" parameters as for `Hash#[]`.

Please, only compare things which are comparable.

Reply:
Agree but keeping it similar to its alias [`Hash#merge!`](https://github.com/JuanitoFatas/fast-ruby/blob/master/code/hash/merge-bang-vs-%5B%5D%3D.rb)

Reply:
Can you please guide me, how to go about it?

Reply:
Thank you @krtschmr As a next step can the Travis CI build be run for this PR and if everything looks good could this PR be considered for next release?